.gitignore

```diff
@@ -34,3 +34,6 @@ Thumbs.db
 
 # Files that can be automatically downloaded that we don't want to ship with our builds
 /src/extensibility/node/node_modules/request/tests/
+
+# .disabled should be ignored, it could be used to disable default extensions
+.disabled
```

Comment:
This should be more specific: `src/extensions/default/*/.disabled`

We may also use this file in unit tests for this feature, and in this case it should be included.